Download document with its attachments (#72)
signnow/SignNowNodeSDK: README.md

#### <a name="download-document"></a>Download a Collapsed Document

> There is a possibility to download a document by original (by default) or archived in zip (set `zip` option to `true`).

Reviewer: ```suggestion
By default document is downloaded without history or attachments. To download it with history set `withHistory` option to `true`. To download it with attachments set `withAttachments` option to `true`.
```

> Also, default document downloads without history. To download it with history set `withHistory` option to `true`.

Reviewer: ```suggestion
```

```javascript
api.document.download({
  token: 'your auth token',
  id: 'document id',
  options: { 

>     zip: true // false by default

Reviewer: ```suggestion
    withAttachments: true, // false by default
```

>     withHistory: true // false by default

Reviewer: ```suggestion
    withHistory: true, // false by default
```

>     }, 

Reviewer: ```suggestion
  },
```
